### PR TITLE
Update jQuery UI 1.10

### PR DESCRIPTION
**Pull request by Culprit-CZ**
In new version are many things deprecated.
We need to upgrade to 1.10 when we upgraded jQuery to 1.9.0 with #204 
Here is [jQuery UI 1.10 upgrade guide](http://jqueryui.com/upgrade-guide/1.10/)
Should be merged with #204 
After jQuery upgrade we become this migration message `JQMIGRATE: jQuery.fn.andSelf() replaced by jQuery.fn.addBack()`

### CODE REVIEW

**Comment by Kamahl19** [comment]
MA

File: root/socialnet/js/m.profile.js [hunk 1]
```diff
@@ -53,30 +53,17 @@
...
 					if (event.which == 3)
 						return false;
-				}).tabs({
-					ajaxOptions: {
-						cache: false,
-						async: false,
-						data: {
-							fullPage: self.tabShowFullPage
-						}
+				}).removeAttr('style').tabs({
```
Comment (senky) [inline root/socialnet/js/m.profile.js]: I am not sure if removing style before triggering tabs is a good step. There will be a while, when elements will be not styled at all.

File: root/socialnet/js/m.profile.js [hunk 2]
```diff
@@ -53,30 +53,17 @@
...
+					beforeLoad: function(e, ui) {
+						ui.ajaxSettings.cache = false;
+						ui.ajaxSettings.async = false;
+						ui.ajaxSettings.url += '&fullPage=' + self.tabShowFullPage;
 					},
-					cache: false,
```
Comment (senky) [inline root/socialnet/js/m.profile.js:64]: You removed it, but I cannot see it redefined elsewhere. Tabs should not be cached, so please include it in ajaxSettigs.

File: root/socialnet/js/m.profile.js [hunk 3]
```diff
@@ -126,7 +113,7 @@
 
 					}
 				})
-				.removeAttr('style');
+
```
Comment (senky) [inline root/socialnet/js/m.profile.js]: Remove this line, please. It is unnecessary doubleline here.

File: root/socialnet/js/m.profile.js [hunk 4]
```diff
@@ -53,30 +53,17 @@
...
-					create: function(e, ui) {
-						if ($sn.getCookie('sn-up-profileTab') == 0) {
-							return false;
-						}
-					},
-					select: function(e, ui) {
-						$(ui.panel).html(self.spinner);
+					beforeActivate: function(e, ui) {
+						ui.newPanel.html(self.spinner);
+						$sn.setCookie('sn-up-profileTab',ui.newTab.index());
```
Comment (senky) [inline root/socialnet/js/m.profile.js]: Space after comma, please:

``` js
$sn.setCookie('sn-up-profileTab', ui.newTab.index());
```

File: root/socialnet/js/m.profile.js (hunk 2 above)
Comment (Culprit-CZ) [inline root/socialnet/js/m.profile.js:64]: Redefined by using `$sn.getCookie` & `$sn.setCookie`

**Comment by senky** [comment]
We can add jQuery UI css to stylesheet.css, so that it gets automatically merged to all other styles and does not consume one extra request for browser.

**Comment by Culprit-CZ** [comment]
We could not. Than we have more work with upgrading.
jQuery UI CSS use: `images/some_image.png`
phpBB CSS use: `{T_THEME_PATH}/images/some_image.png`

If we do, then we must have in mind the editing CSS.
Thus, we allow a simple change of style with [jQuery UI ThemeRoller](http://jqueryui.com/themeroller/)

**Comment by Kamahl19** [comment]
Looks good, MA

**Comment by senky** [comment]
MA

**Pull request merged**